client/state/selectors/get-selected-editor.js

```diff
@@ -0,0 +1,19 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+
+/**
+ * Returns the editor of the selected site
+ * @param {Object} state Global state tree
+ * @param {Number} siteId Site ID
+ * @return {String} "gutenberg" or "classic" or null
```

Comment:
why `null`? let's explain

probably it means "we haven't loaded the information yet"